universal-application-tool-0.0.1/app/views/admin/programs/ProgramAdministratorProgramListView.java

```diff
@@ -31,7 +31,7 @@
 
   @Inject
   public ProgramAdministratorProgramListView(AdminLayout layout, Config config) {
-    this.layout = checkNotNull(layout);
+    this.layout = checkNotNull(layout).setProgramAdminType();
```

Comment:
This can be shown to a global admin too, actually, since the global admin is the admin of every program with no admin.

Reply:
Agree it will be better if we can switch based on roles.

I also wonder if this is a change across different admins, as layout is not re-created for each request.

Reply:
> I also wonder if this is a change across different admins, as layout is not re-created for each request.

I'm not following this part of our comment

Reply:
Resolved in chat, layout objects are not re-used between requests.